toody-frontend/src/pages/Home/Home.jsx

```diff
@@ -17,12 +17,19 @@ const Home = ({ name }) => {
     // TODO: Complete this function with an axios.get() call
     // to the appropriate backend endpoint for todos. The todos on
     // the homepage should populate correctly when this is done.
+    // const [people, setPeople] = useState([]);
+    const { data } = await axios.get('http://localhost:3001/todos/');
+    setTodos(data);
+    // todos.map(todosResult => <h1> {todosResult.text}</h1>);
```

Comment:
in general, extra lines/comments/`console.log`s should be removed once our code is ready for review - not sure if this was already here but i don't think it should be